sklearn/cluster/optics_.py

```diff
@@ -511,7 +538,8 @@ def cluster_optics_dbscan(reachability, core_distances, ordering, eps=0.5):
 
     eps : float, optional (default=0.5)
         DBSCAN `eps` parameter. Must be set to < `max_eps`. Results
-        will be close to DBSCAN algorithm if `eps` is < (`max_eps` / 5)
+        will be close to DBSCAN algorithm if ``eps`` and ``max_eps`` are close
+        to one another.
```

Comment:
I don't see why they need to be close. I understand that `eps` can't be larger than `max_eps`, and that it can be problematic if it is close. But assume max_eps=inf?

Reply:
This is why I changed the description as such: https://github.com/scikit-learn/scikit-learn/pull/12077#issuecomment-476191088

Do you disagree?

Reply:
I am not sure what factor that comment refers to, and whether that test makes sense. From a quick look, it requires labels to be identical, but should allow permutation of labels (except for noise).